tests/cli/test_config.py

```diff
@@ -722,3 +716,194 @@ def test_custom_multichannels_prepend_duplicate(conda_cli: CondaCLIFixture):
             == "Warning: 'bar' already in 'custom_multichannels.foo' list, moving to the top"
         )
         assert _read_test_condarc(rc) == custom_multichannels_expected
+
+
+def test_conda_config_describe(
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+    monkeypatch: MonkeyPatch,
+):
+    RE_PARAMETERS = (
+        re.compile(rf"^# # {name} \(", flags=re.MULTILINE)
+        for category, names in context.category_map.items()
+        if category not in ("CLI-only", "Hidden and Undocumented")
```

Comment:
Not a huge fan of this hard-coding..

Reply:
you're referring to `("CLI-only", "Hidden and Undocumented")`?

yeah I agree, I'm also not a fan of how we define or handle the condarc variable descriptions